action.yaml

```diff
@@ -15,10 +15,21 @@ inputs:
   PREFECT_OSS_ACTIONS_RW_TOKEN:
     description: The Github token to use for triggering the Prefect OSS NPM package update.
     required: false
-  UPDATE_DOWNSTREAM_REPOS:
-    description: Whether to trigger the Nebula UI and Prefect OSS NPM package updates.
+  PREFECT_UI_LIBRARY_ACTIONS_RW_TOKEN:
```

Comment:
for my knowledge - does `RW` stand for read/write?

Reply:
yessir

Reply:
great, thanks!